### PR TITLE
Chore upgrade to ckan 2.8

### PR DESCRIPTION
**Pull request by esirK**
## Description

Upgrades to ckan 2.8

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: .env.tmpl [hunk 1]
```diff
@@ -12,10 +12,10 @@ CKAN___BEAKER__SESSION__SECRET=
 CKAN___APP_INSTANCE_UUID=
 
 # Database config
-CKAN__DATASTORE_WRITE_URL=postgres://ckan_default:pass@db/datastore_default
-CKAN__DATASTORE_READ_URL=postgres://datastore_default:pass@db/datastore_default
+CKAN_DATASTORE_WRITE_URL=postgres://ckan_default:pass@db/datastore_default
+CKAN_DATASTORE_READ_URL=postgres://datastore_default:pass@db/datastore_default
 
-CKAN__REDIS__URL=redis://redis:6379/0
+CKAN_REDIS_URL=redis://redis:6379/0
```
Comment (kilemensi) [inline .env.tmpl:18]: Why are we changing the `__` to `_`?

File: README.md [hunk 2]
```diff
@@ -14,16 +14,14 @@ We use CKAN's own vanilla releases but because they haven't properly adopted Doc
 The ckan extensions we are using include:
 
 - ckanext-openafrica - https://github.com/CodeForAfrica/ckanext-openafrica
-- ckanext-datarequests - https://github.com/CodeForAfricaLabs/ckanext-datarequests
+- ckanext-datarequests - https://github.com/conwetlab/ckanext-datarequests
 - ckanext-harvester - https://github.com/ckan/ckanext-harvest
-- ckanext-socialite (experimental) - https://github.com/CodeForAfricaLabs/ckanext-socialite
-- ckanext-social - https://github.com/CodeForAfricaLabs/ckanext-social
-- ckanext-notify - https://github.com/CodeForAfricaLabs/ckanext-notify
```
Comment (kilemensi) [inline README.md:21]: Are we no longer using these 3 extensions?

File: contrib/ckan/meta.py [hunk 3]
```diff
@@ -0,0 +1,147 @@
+# encoding: utf-8
+
+import datetime
```
Comment (kilemensi) [inline contrib/ckan/meta.py:3]: Lets add a big comment here why we need this file (and if it should be removed on future upgrades e.g. to 2.9).

**Review by kilemensi — COMMENTED** [review]
👍🏽

**Review by thepsalmist — COMMENTED** [review]
LGTM @esirK , except the comment on `meta.py` as suggested by clemence

File: README.md (hunk 2 above)
Comment (esirK) [reply README.md:21]: yeah. They were not even being used.

File: .env.tmpl (hunk 1 above)
Comment (esirK) [reply .env.tmpl:18]: I think this is what the ckan code expects; not sure why we had the `__`

**Pull request merged**